providers/providers.go

```diff
@@ -23,6 +23,8 @@ type Provider interface {
 	ValidateRequest(*http.Request) (*SessionState, error)
 	GetLoginURL() (*url.URL, error)
 	GetRedeemURL() (*url.URL, error)
+	SetReviewCAs([]string)
```

Comment:
You do not use this in a generic context so the methods do not need to be part of the provider interface.

Reply:
done